src/cli/load.test.js

```diff
@@ -544,7 +553,27 @@ describe("cli/load", () => {
         repoIds: [fooBar, fooBaz],
       });
       expect(saveCred).toHaveBeenCalledTimes(1);
-      expect(saveCred).toHaveBeenCalledWith(graph, fooCombined);
+      expect(saveCred).toHaveBeenCalledWith(graph, fooCombined, undefined);
+    });
+
+    it("calls saveCred with the weight path if set in the options", async () => {
```

Comment:
Along the lines of the previous test case, I've included the case where a path _is_ set.